src/rez/tests/test_config.py

```diff
@@ -39,11 +39,13 @@ def _test_overrides(self, c):
         c.override("build_directory", "floober")
         c.override("plugins.release_vcs.tag_name", "bah")
         c.override("plugins.release_hook.emailer.sender", "joe.bloggs")
+        c.override("implicit_packages", ["a list", "of values"])
```

Comment:
This fails without the keep_plugins fix. The reason that the tests pass is because a plugin override happens to be the trailing override. But if another override happens afterwards with will clear the plugin overrides.